README.md

```diff
@@ -1 +1,134 @@
-# prometheus_alertmanager
\ No newline at end of file
+# prometheus_alertmanager
+
+Master: [![Build Status](https://travis-ci.org/sansible/prometheus_alertmanager.svg?branch=master)](https://travis-ci.org/sansible/prometheus_alertmanager)  
+Develop: [![Build Status](https://travis-ci.org/sansible/prometheus_alertmanager.svg?branch=develop)](https://travis-ci.org/sansible/prometheus_alertmanager)
+
+* [ansible.cfg](#ansible-cfg)
+* [Installation and Dependencies](#installation-and-dependencies)
+* [Tags](#tags)
+* [Examples](#examples)
+
+This roles installs Prometheus Alert Manager.
+
+Prometheus Alert Manager sends alerts on behalf of Prometheus server.
+
+For more information about Prometheus Alert Manager please visit:
+[https://github.com/prometheus/alertmanager](https://github.com/prometheus/alertmanager).
+[https://prometheus.io/docs/alerting/alertmanager/](https://prometheus.io/docs/alerting/alertmanager/).
+
+For more information about Prometheus please visit:
+[https://prometheus.io/docs/](https://prometheus.io/docs/).
+
+
+## ansible.cfg
+
+This role is designed to work with merge "hash_behaviour". Make sure your
+ansible.cfg contains these settings
+
+```INI
+[defaults]
+hash_behaviour = merge
+```
+
+## Installation and Dependencies
+
+This role will install `sansible.users_and_groups` for managing `prometheus_server` user.
```

Comment:
Need to change prometheus_server to prometheus_alertmanager in a few places here.